### PR TITLE
[Validator] add validation message translations for the WordCount constraint

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

### CODE REVIEW

**Review by alexandre-daubois — APPROVED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf [hunk 1]
```diff
@@ -442,6 +442,14 @@
                 <source>This URL is missing a top-level domain.</source>
                 <target state="needs-review-translation">Die URL mis 'n topvlakdomein.</target>
             </trans-unit>
+            <trans-unit id="114">
+                <source>This value is too short. It should contain at least one word.|This value is too short. It should contain at least {{ min }} words.</source>
```
Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.af.xlf:446]: In English, we should have a specific case for when min == 1, right?

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Component/Validator/Resources/translations/validators.af.xlf:446]: Isn’t this the part on the left side of the `|`?

Comment (fabpot) [reply src/Symfony/Component/Validator/Resources/translations/validators.af.xlf:446]: I missed the |, sorry

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**